packages/cognito-custom-mail-lambda/package.json

```diff
@@ -7,8 +7,9 @@
     "dist"
   ],
   "scripts": {
+    "generate-html": "yarn ts-node src/scripts/generate.ts",
```

Comment:
Why not just put this in the build step then you don't need to commit the html and you know you always have the latest compiled templates before releasing?

Reply:
yea guess so, needed them for tests 😭